README.md

```diff
@@ -43,14 +67,20 @@ Click [here](https://werthdavid.github.io/kjua/)
 * `fontname` font for additional label text (defaults to `sans-serif`)
 * `fontcolor` font-color for additional label text (defaults to `#333`)
 * `image` additional image (defaults to `undefined`, use an HTMLImageElement or base64-string)
+* `imageAsCode` draw the image as part of the code (defaults to `false`)
 
 More details can be found on [larsjung.de/kjua](https://larsjung.de/kjua/)
 
 ## Differences to kjua
 
-This is basically just a fork of kjua that adds the possibility to render QR-codes as SVG.
+* possibility to render QR-codes as SVG
+* transparent background support
```

Comment:
And how can we use that?

Reply:
Transparent background? Just provide null/undefined or an empty string as „back“ option